lib/_tools.scss

```diff
@@ -76,14 +75,10 @@
         -webkit-flex-flow: row wrap;
         -ms-flex-flow: row wrap;
         flex-flow: row wrap;
-        -webkit-flex: 1 1 auto; // [2]
-        -ms-flex: 1 1 auto; // [2]
-        flex: 1 1 auto; // [2]
-
-        margin-right: 0; // [3]
+        margin-right: 0; // [2]
```

Comment:
OK. I thought that for boxes [1][2][3] in ltr, when in rtl, they'd be [3][2][1], but if 1 had a `margin-left: 8px` on box 1, it would remain on the left.